core/lib/constants/src/contracts.rs

```diff
@@ -120,3 +120,8 @@ pub const MINT_AND_BURN_ADDRESS: H160 = H160::zero();
 
 // The `storage_log.value` database value for a contract that was deployed in a failed transaction.
 pub const FAILED_CONTRACT_DEPLOYMENT_BYTECODE_HASH: H256 = H256::zero();
+
+pub const ETHEREUM_SHARED_BRIDGE_ADDRESS: Address = H160([
```

Comment:
A doc-comment here would be appreciated.

Reply:
Done, wdy?